public/css/style.css

```diff
@@ -41,14 +41,19 @@ nav {
   justify-content: space-around;
 }
 
-.currentActor {
-  background-color: #c9ada7;
-  border-top-right-radius: 15px;
-  border-bottom-right-radius: 15px;
+#scriptDiv pre {
+  border-left-color: #d1d1d1;
+  padding: 1rem 1.5rem;
+  background-color: inherit;
+  font-family: Roboto, serif;
+  margin: 0 0 10px 0
```

Comment:
I added a couple styles from `<blockquote>` so `<pre>` would be able to replace it.